File: array/python/Delete.py

number_array = [4, 2, 7, 9, 13,5]
print("=====Initial Array====")
for index, value in enumerate(number_array):
    print(index, value)

for index,enumerated_number in enumerate(number_array):
    if enumerated_number == 5:
        print("Found 5 at index:", index)
        del number_array[index]
        print("=====After Delete====")
print(number_array)

File: array/c/Delete.c
#include <stdio.h>
int main()
{
    int LA[7] = {5, 10, 15, 20, 25}, N = 5, K = 3, item;
    item = LA[K];
    int J;
    for (J = K; J <= N - 1; J++)
    {
        LA[J] = LA[J + 1];
    }

    N = N - 1;
    printf("Item delete is %d\n", item);
    int i;
    for (i = 0; i < N; i++)
    {
        printf("%d ", LA[i]);
    }
    return 0;
}
